### PR TITLE
fixes #421, each test suite have their own skip table

### LINKED ISSUE #421: Rework failed tests skip table for each test.
Currently, all tests using `test_helpers` also using it's skip table. need to split this table for each of distinct test. minimize confusion.

### PR DESCRIPTION
**Pull request by jangko**
less confusion about which test suite should skip what tests

### CODE REVIEW

File: tests/test_helpers.nim [hunk 1]
```diff
@@ -171,7 +65,7 @@ proc lacksSupportedForks*(fixtures: JsonNode): bool =
 
 var status = initOrderedTable[string, OrderedTable[string, Status]]()
 
-macro jsonTest*(s: static[string], handler: untyped): untyped =
+macro jsonTest*(s: static[string], handler: untyped, skipTest: untyped): untyped =
```
Comment (stefantalpalaru) [inline tests/test_helpers.nim]: We can't give `skipTest` the default value of `skipNothing`?

Comment (jangko) [reply tests/test_helpers.nim]: sure we can. my memory must be corrupted when I am not using default value for `skipTest`.
fixed.

**Comment by zah** [comment]
It's hard to tell whether the CI failure here is genuine or not. I'll close and re-open to give it another try.

**Comment by jangko** [comment]
I could not reproduce the problem running archlinux in a virtualbox.

**Comment by stefantalpalaru** [comment]
Could it be a lingering server blocking a port? https://travis-ci.org/status-im/nimbus/jobs/615173024#L7175

Maybe we don't wait for it to close properly before starting the next test suite.

**Comment by kdeme** [comment]
hmm, strange that with that error those rpc tests actually manage to pass, no?

**Comment by jangko** [comment]
last time I saw both of travis CI failed, now it's only one of them. could it be another hiccup?
who can restart the failed CI?

**Comment by stefantalpalaru** [comment]
Restarted.

**Comment by jangko** [comment]
finally tests are green.

**Pull request merged**